dashboard/src/modules/components/ComparisonComponent/index.jsx

```diff
@@ -19,11 +19,17 @@ import {
 } from "actions/comparisonActions";
 import { useDispatch, useSelector } from "react-redux";
 
+import Cookies from "js-cookie";
 import PanelConent from "./PanelContent";
 import { getDatasets } from "actions/overviewActions";
 
 const ComparisonComponent = () => {
   const dispatch = useDispatch();
+<<<<<<< HEAD
+=======
+  const navigate = useNavigate();
+  const loggedIn = Cookies.get("isLoggedIn");
+>>>>>>> 1e8b2dc6c (public datasets list)
```

Comment:
I'm assuming that the lint will fail because you've left in this merge conflict. It needs to be resolved!

Reply:
Looks like this has been fixed in the latest commit.